samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb

```diff
@@ -9,15 +9,15 @@
     "In this notebook you will create e2e mnist Kubeflow Pipeline to perfom:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.    run_id = kfp_client.create_run_from_pipeline_func(mnist_pipeline, namespace=namespace, arguments={},service_account='default-editor').run_id
</pre><p>I know <code>service_account='default-editor'</code> was needed in our environment, but does it make sense to add this in general? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/7355/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
@zijianjoy Does it make sense to be explicit with the service account here?

Reply:
@pvaneck @zijianjoy is `service_account='default-editor' ` a temporary thing, It wasn't required in previous versions that I tried out

Reply:
I think service_account can be omitted, but you need to use KFP 1.8.1 to fix it. Reference: https://github.com/kubeflow/pipelines/pull/7337